### ISSUE TITLE
proposing django-admin-bootstrapped

### ISSUE THREAD

**Issue by xrmx** [jazzband/roadies#15]
Hello, i'd like to propose moving django-admin-bootstrapped [0] to jazzband organization. The app is in good shape but i don't have much motivations to keep it updated (django 1.9 support is missing atm) as i don't have much use for it anymore.
Looking at the guidelines we should be ok with everything, some minor nits like moving the contributing section we have on our README.md to a separated files and jazzband tags and CoC links.

[0] https://github.com/django-admin-bootstrapped/django-admin-bootstrapped

**Comment by jezdez** [comment]
@xrmx Yep, looks good to me, too. Let me know when you want to do the switch, so I can enable Travis etc.

**Comment by xrmx** [comment]
Il 14/04/2016 10:10, Jannis Leidel ha scritto:
> @xrmx <https://github.com/xrmx> Yep, looks good to me, too. Let me know when you
> want to do the switch, so I can enable Travis etc.

I can transfer the repository tomorrow if you are fine with that. If you add me 
to the organization i'll split contruting and add proper badges and CoC 
references too. Thanks!

**Comment by jezdez** [comment]
@xrmx Hmmm, I don't think you've read the Jazzband docs if you're asking to be added to the organization. You add yourself to the GitHub organization by joining on jazzband.co. I'm a bit weary now to move the app to the Jazzband if you haven't seen that central feature of the Jazzband to be honest, can you elaborate how you found Jazzband? Have you read and understood the guidelines?

**Comment by xrmx** [comment]
@jezdez I've read the docs i've just missed the "Join Jazzband" link until now sorry, making the "join" in the first paragraph a link may have helped though :) Anyway I've read the guidelines as you can see i've made references to them in my proposal.
